php/Exchange.php

```diff
@@ -642,6 +643,14 @@ public static function is_json_encoded_object ($input) {
                 (($input[0] === '{') || ($input[0] === '['));
     }
 
+    public static function is_array ($object) {
+        return is_array($object);
```

Comment:
In fact, you've introduced one here...
Because in PHP `$object == $array`.
See, how we transpile js `Array.isArray ()` here: 

- https://github.com/ccxt/ccxt/blob/master/transpile.js#L143
- https://github.com/ccxt/ccxt/blob/master/transpile.js#L258

We will have to fix this before merging. Namely, change all instances of `this.isArray` to `Array.isArray` or change the implementation in php.